test/util/images/images.go

```diff
@@ -0,0 +1,41 @@
+package images
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+var registry string
+var images map[string]imageLocation
+
+func init() {
+	registry = os.Getenv("IMAGE_REGISTRY")
+
+	images = map[string]imageLocation{
+		"testutils": {
+			name:    "cnftest-utils",
+			registy: "quay.io/fpaoline/",
```

Comment:
Do we have a dockerfile for this anywhere?

Reply:
Yes, all the dockerfiles are [here](https://github.com/openshift-kni/cnf-features-deploy/tree/master/cnf-tests/testimages) and we are in the process of mirroring those images under the openshift-kni quay repo, which is the reason of the hold